Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -75,6 +75,28 @@ open class ChartDataSet: ChartBaseDataSet
     
     /// minimum x-value in the value array
     internal var _xMin: Double = Double.greatestFiniteMagnitude
+
+    /// *
+    /// - note: Calls `reloadData()` after setting a new value.
+    /// - returns: The array of y-values that this DataSet represents.
+    @objc open var values: [ChartDataEntry]
```

Comment:
This variable no longer exists. Please remove and update the `didSet` method on the new variable of the same name.